mod_test/controllers.py

```diff
@@ -47,7 +48,6 @@ def index():
         'TestType': TestType
     }
 
-
```

Comment:
Line must not be removed for PEP-8 compliancy. I urge you to edit the files with a full blown IDE (e.g. PyCharm, they have a free community edition)